fbw-a380x/mach.config.js

```diff
@@ -32,10 +32,11 @@ module.exports = {
         msfsAvionicsInstrument('ND'),
         msfsAvionicsInstrument('PFD'),
         msfsAvionicsInstrument('RMP'),
+        msfsAvionicsInstrument('EWDv2'),
```

Comment:
Please keep alphabetical. It lowers the chance of merge conflicts.

Reply:
fixed